src/interceptors/index.ts

```diff
@@ -2,10 +2,13 @@ import * as xhrInterceptor from './xhrInterceptor';
 import * as fetchInterceptor from './fetchInterceptor';
 
 export interface Interceptors {
-    XMLHttpRequest: typeof xhrInterceptor;
-    fetch: typeof fetchInterceptor;
+  XMLHttpRequest: typeof xhrInterceptor;
+  fetch: typeof fetchInterceptor;
+  http?: any;
+  https?: any;
```

Comment:
This is needed, because we allows these values to be in the list of `strategies`